software/control/_def.py

```diff
@@ -230,6 +230,10 @@ class CAMERA_CONFIG:
     ROI_OFFSET_Y_DEFAULT = 0
     ROI_WIDTH_DEFAULT = 3104
     ROI_HEIGHT_DEFAULT = 2084
+    BINNING_FACTOR_DEFAULT = (2, 2)
```

Comment:
This should be `(1,1)`. Since user needs to update the ini file to make the pixel size correct, they can add `BINNING_FACTOR_DEFAULT = (2, 2)` to the configuration file